src/Model.php

```diff
@@ -65,28 +65,30 @@
 	 */
 	public $exists = false;
 
-	// TODO implement timestamps
+	/**
+	 * The name of the "created at" column.
+	 *
+	 * @var string
+	 */
+	const CREATED_AT = 'create_time';
+
+	/**
+	 * The name of the "updated at" column.
+	 *
+	 * @var string
+	 */
+	const UPDATED_AT = 'update_time';
 
 	/**
 	 * Create a new Joomla entity model instance.
 	 *
-	 * @param   array $attributes -> preloads any attributed for the model
+	 * @param   DatabaseDriver $db         database driver instance
+	 * @param   array          $attributes -> preloads any attributed for the model
 	 */
-	public function __construct(array $attributes = array())
+	public function __construct($db, array $attributes = array())
```

Comment:
Add an explicit typehint of `DatabaseDriver` in the code here

Reply:
is this fine? Passing the DatabaseDriver to the constructor? Is there any other way to do it? I would like to keep the static method calls usage like this: `User::findLast()`. Otherwise, I would have to pass the DatabaseDriver for every static method call.